From 99cc28e0eb1a37233c0f7d3595175d1f331550e7 Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Tue, 1 Mar 2022 15:56:30 +0100
Subject: [PATCH 1/5] build: Treat --with-x[=yes] the same as --with-x=auto

Previously, --with-x would disable the check for X11 libraries, which
was not intended.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index fa75e930..847e95fb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1115,7 +1115,7 @@ if test "x$dbus_win" = xyes; then
 
     enable_x11_autolaunch=no
     have_x11=no
-else if test "x$with_x" = xauto; then
+else if test "x$with_x" != xno; then
     PKG_CHECK_MODULES([X], [x11],
             [AC_DEFINE([HAVE_X11], [1], [Define to 1 if you have X11 library])],
             [ have_x11=no ])
-- 
2.36.0

